integration_test.go

```diff
@@ -101,10 +151,12 @@ func testRequestExistingRepo(t *testing.T) {
 	if !os.IsNotExist(err) {
 		t.Error("repo1/test.db should be cached")
 	}
+
 }
 
 func testRequestExistingRepoWithDb(t *testing.T) {
 	// Requesting existing repo
+
```

Comment:
I see you added a number of newlines. Is it really needed for this pull request?

Reply:
Nope, don't remember if it was due to the vscode linter or if i had added them myself